locale/es/ManageIQ_Api.po

```diff
@@ -1,27 +1,28 @@
-# Spanish translations for ManageIQ_Api package.
-# Copyright (C) 2018 THE PACKAGE'S COPYRIGHT HOLDER
```

Comment:
Was this supposed to change?

Reply:
This is OK I suppose. These comments don't matter that much really, those removed lines were generated automatically during string collection.